libyara/modules/pe.c

```diff
@@ -1961,6 +1999,7 @@ begin_declarations;
   declare_function("imports", "s", "i", imports_dll);
   declare_function("locale", "i", "i", locale);
   declare_function("language", "i", "i", language);
+  declare_function("rich_version", "i", "i", rich_version);
```

Comment:
I would put this function inside the "rich_signature" structure and rename it to just "version". The rule would looks like: pe.rich_signature.version(21005).

And what about checking for tool IDs, or IDs/version pairs? Does it makes sense for you?